src/test/groovy/nebula/plugin/publishing/ivy/interaction/IvyPublishRecommenderInteractionSpec.groovy

```diff
@@ -18,7 +18,7 @@ class IvyPublishRecommenderInteractionSpec extends IntegrationTestKitSpec {
 
         buildFile << """\
             plugins {
-                id 'java'
+                id 'java-library'
```

Comment:
Should we keep both flavors? `java` and `java-library`?

Reply:
sure it doesn't hurt, added it in https://github.com/nebula-plugins/nebula-publishing-plugin/pull/129/commits/e18c9527cef9bae62510efde2ad47e158e3cc3d4